provider/cmd/pulumi-resource-frontegg/main.go

```diff
@@ -9,21 +11,23 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+//go:generate go run ./generate.go
+
 package main
 
 import (
 	_ "embed"
 
-	frontegg "github.com/MaterializeInc/pulumi-frontegg/provider"
 	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	frontegg "github.com/MaterializeInc/pulumi-frontegg/provider"
+	"github.com/MaterializeInc/pulumi-frontegg/provider/pkg/version"
 )
 
-// Injected by linker in release builds.
-var version string
-
-//go:embed schema.json
+//go:embed schema-embed.json
```

Comment:
the schema file looks like it's named `schema.json` - so is this embedding the correct file?

Reply:
I don't really know what's going on here, but I believe it's something like it reads the schema.json file and writes a schema-embed.json file with an extra version key... since this key/file are generated they're gitignored. 

https://github.com/MaterializeInc/pulumi-frontegg/pull/14/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R20